src/scss/skin-modern/_skin-ads.scss

```diff
@@ -28,16 +28,6 @@
         }
       }
 
-      /* Let button look like it is not click-able during the skippable-in countdown */
```

Comment:
This idicates that the branching base was wrong (branch from #202 instead of develop).